cmake/modules/RootMacros.cmake

```diff
@@ -469,9 +580,9 @@ function(ROOT_GENERATE_DICTIONARY dictionary)
     unset(cpp_module_file)
   endif()
 
-  if(CMAKE_ROOTTEST_NOROOTMAP OR cpp_module_file)
-    set(rootmap_name )
-    set(rootmapargs )
+  if(CMAKE_ROOTTEST_NOROOTMAP OR cpp_module_file OR ARG_MULTIDICT)
```

Comment:
In case of `ARG_MULTIDICT` we still generate rootmap files. You could check Smatrix and GenVector.

Reply:
> You could check Smatrix and GenVector.

There are libGenVector32.rootmap and libSmatrix32.rootmap files.
But I did not found `libGenVector32` or `libSmatrix32` libraries. 
Are these rootmap files required?

Reply:
Yes, they are, we have separate dictionaries for `Double32_t` and `double` as we use `Double32_t` as a conditional strong typedef. We need to generate different instantiations for both and store the ones with `Double32_t` as floats on disk...

Reply:
I already revert this change.